MAUI/DataGrid/loadMore.md

```diff
@@ -0,0 +1,195 @@
+
+When the grid reaches maximum offset while scrolling down, an interactive DataGridLoadMoreView is displayed. Tapping the load more view triggers a command to add more data to the data source of the grid at runtime.
+
+## Load more command
+The DataGrid loads records to its data source at runtime by triggering an ICommand bound to the `SfDataGrid.LoadMoreCommand` property. When the DataGridLoadMoreView is tapped, the CanExecute of the ICommand returns true, and this command is triggered to load the records at runtime.
+
+Set the `SfDataGrid.IsBusy` property to true before loading items to notify the grid that more items are to be loaded. Set the property to false after loading items into the grid. When loading items, alter the time for the LoadMore animation from the sample by setting a delay based on the requirement.
+
+To enable and load items at runtime, follow the code example:
+
+```C# 
+//Enable load more in SfDataGrid
+dataGrid.AllowLoadMore = true;
+dataGrid.LoadMoreCommand = new Command(ExecuteLoadMoreCommand);
+ 
+private async void ExecuteLoadMoreCommand()
+{
+    this.dataGrid.IsBusy = true;
+    await Task.Delay(new TimeSpan(0, 0, 5));
+    viewModel.LoadMoreItems();
+    this.dataGrid.IsBusy = false;
+} 
+
+//ViewModel.cs
+public OrderInfoRepository()
+{
+    orderInfo = this.GenerateOrders(25);
+}
+
+
+//ItemsSource
+private ObservableCollection<OrderInfo> orderInfo;
+public ObservableCollection<OrderInfo> OrderInfoCollection
+{
+    get { return orderInfo; }
+    set { this.orderInfo = value; }
+}
+
+//ItemsSource Generator
+public ObservableCollection<OrderInfo> GenerateOrders(int count)
+{
+    ObservableCollection<OrderInfo> orders = new ObservableCollection<OrderInfo>();
+    for (int i = 1000; i <= 1000 + count; i++)
+    {
+        var order = new OrderInfo(i, 
+            CustomerID[random.Next(7)],
+            Country[random.Next(7)], 
+            Customer[random.Next(7)], 
+            ShipCity[random.Next(7)]);
+        orders.Add(order);
+    }
+    return orders;            
+}
+
+public void LoadMoreItems()
+{
+    for (int i = 0; i < 10; i++)
+        this.OrderInfoCollection.Add(this.GenerateOrderInfo(OrderInfoCollection.Count));
+}
+
+
+public OrderInfo GenerateOrderInfo(int count)
+{
+    var order = new OrderInfo(
+        count + 1000, CustomerID[random.Next(7)],
+        Country[random.Next(7)],
+        Customer[random.Next(7)],
+        ShipCity[random.Next(7)]);
+
+    return order;
+}
+```
+![DataGridLoadMoreView](Images\loadmore\maui-datagrid-loadmoreview.gif)
+
+## Customize DataGridLoadMoreView
+
+### Customize DataLoadMoreView display text
+Customize the text displayed in the `LoadMoreView` by setting the `SfDataGrid.LoadMoreText` property as follows:
```

Comment:
The load more text can be customized by using the `SfDataGrd.LoadMoreText` property. The default value of this property is "Load More".

Reply:
Removed this topic